benchmark/src/main/scala/code/winitzki/benchmark/Benchmarks9.scala

```diff
@@ -33,7 +33,7 @@ object Benchmarks9 {
 
     val tp = new FixedPool(threads)
```

Comment:
make_ping_pong_stack and some other methods have client of benchmark create the thread pool. Would you consider Benchmarks* methods receive a tp rather than a threads parameter? Philosophy of threaded code in Scala (and more generally inversion of control) is to pass an ExecutionContext as implicit parameter so the underlying method is not responsible for constructing that machinery.